quandl/get_table.py

```diff
@@ -11,6 +11,11 @@ def get_table(code, **options):
     else:
         paginate = None
 
+    if 'returns' in options.keys():
+        returns = options.pop('returns')
```

Comment:
duplicate code, let's put this into a utility method so that it can be shared with `quandl.get`